File: java/terminal/terminal-core/src/main/java/com/github/fernthedev/terminal/core/CandidateUtil.java
package com.github.fernthedev.terminal.core;

import com.github.fernthedev.lightchat.core.data.LightCandidate;
import org.jline.reader.Candidate;

public class CandidateUtil {

    public static Candidate toCandidate(LightCandidate lightCandidate) {
        return new Candidate(lightCandidate.getValue(),lightCandidate.getDispl(),lightCandidate.getGroup(),lightCandidate.getDescr(),lightCandidate.getSuffix(),lightCandidate.getKey(),lightCandidate.isComplete());
    }

}

File: java/terminal/terminal-core/src/main/java/com/github/fernthedev/terminal/core/CandidateUtil.kt
package com.github.fernthedev.terminal.core

import com.github.fernthedev.lightchat.core.data.LightCandidate
import org.jline.reader.Candidate

object CandidateUtil {
    @JvmStatic
    fun toCandidate(lightCandidate: LightCandidate): Candidate {
        assert(lightCandidate.value != null)
        return Candidate(
            lightCandidate.value,
            lightCandidate.displ,
            lightCandidate.group,
            lightCandidate.descr,
            lightCandidate.suffix,
            lightCandidate.key,
            lightCandidate.isComplete
        )
    }
}